### PR TITLE
Update metadata.yaml

### PR DESCRIPTION
**Pull request by naglem**
Add author affiliations

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -70,6 +70,14 @@ authors:
...
       - Department of Pathology, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA
     funders: GBMF4552
+  - github: naglem
+    name: Michael W. Nagle
+    initials: MWN
+    orcid: 0000-0002-4677-7582
+    twitter: mikenagle84
+    email: mike_nagle@eisai.com
+    affiliations:
+      - Neurogenomics, Translational Sciences, Neurology Business Group, Eisai Inc, Cambridge, Massachusetts, United States of America
```
Comment (dhimmel) [inline content/metadata.yaml:80]: Should we also add your Pfizer affiliation that was active during this research?

**Review by dhimmel — COMMENTED** [review]
Thanks @naglem! Hope things are well.

File: content/metadata.yaml (hunk 1 above)
Comment (naglem) [reply content/metadata.yaml:80]: Yeah, would probably be appropriate. I can modify and resend the request

**Comment by dhimmel** [comment]
superseded by https://github.com/greenelab/connectivity-search-manuscript/pull/36

**Pull request closed without merging**